src/Bathymetry.jl

```diff
@@ -132,27 +106,15 @@ function regrid_bathymetry(target_grid;
         return throw(ArgumentError("interpolation_passes has to be an integer ≥ 1"))
     end
 
-    filepath = download_bathymetry(; url, dir, filename)
+    filepath = metadata_path(m)
     dataset = Dataset(filepath, "r")
 
+    grid = native_grid(m, architecture(target_grid); halo = (10, 10, 1))
```

Comment:
This file can't seem to 'see' the `native_grid` methods I have imported, I have no idea why!